gf-webapp-admin/src/component/rooms/room/services/Services.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import {BASE_SERVICES} from "../../../../utils/consts";
+import {EditBtn} from "../editBtn/EditBtn";
+import styles from "./services.module.scss";
+
+export const Services = ({room}:{room:IRoom}) => {
+
+  const roomServices = room.services?.split(",").map(el => el.trim());
+  const [services, setServices] = useState(roomServices);
+  const [isDisabled, setIsDisabled] = useState(true);
+  const baseServices = BASE_SERVICES;
+
+  const saveHandler = async () => {
+    setIsDisabled(true);
+    room.services = services?.join(", ").trim();
+    saveRoom(room);
+  };
+
+  const disabledHandler = () => {
+    setIsDisabled(false);
+  };
+
+  const handleChange = (e:React.ChangeEvent) => {
+    const target = e.target as HTMLInputElement;
+    setServices(prev => prev?.indexOf(target.value) === -1 ?
```

Comment:
nice) Refactoring required for setServices argument

Reply:
Done. Add function `changeRoomServices` with correct naming